### PR TITLE
build/auto deploy on production

### PR DESCRIPTION
**Pull request by new5558**

### CODE REVIEW

File: .github/workflows/build-and-deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+on:
+  push:
+    branches: [master]
+jobs:
+  build-and-deploy:
+    strategy:
+      matrix:
+        node-version: [14.x]
+        platform: [ubuntu-latest]
+    runs-on: ${{ matrix.platform }}
```
Comment (bombnp) [inline .github/workflows/build-and-deploy.yaml]: what's the point of using strategy if we're gonna use only one strategy?

Comment (bombnp) [reply .github/workflows/build-and-deploy.yaml]: just curious

Comment (new5558) [reply .github/workflows/build-and-deploy.yaml]: fixed

File: .github/workflows/build-and-deploy.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Install dependencies
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: yarn --fronzen-lockfile
+
+      - name: Build 🔧
+        run: |
+          yarn build
+          yarn export
```
Comment (bombnp) [inline .github/workflows/build-and-deploy.yaml:30]: what does this do? i've never seen it

Comment (new5558) [reply .github/workflows/build-and-deploy.yaml:30]: When next js build the project, It built server-side rendering built and has to be exported to static site using command `next export`

**Review by bombnp — APPROVED** [review]

**Pull request merged**